src/algorithms/banditpam.cpp

```diff
@@ -104,6 +108,9 @@ namespace km {
       }
       updated_sigma(i) = arma::stddev(sample);
     }
+    // reset batchSize the original batch size as it's a global variable
```

Comment:
sorry I should have added "to" between "batchSize" and "the original batch size"